integration-tests/pom.xml

```diff
@@ -272,6 +272,7 @@
                 <module>native-config-profile</module>
                 <module>logging-min-level-unset</module>
                 <module>logging-min-level-set</module>
+                <module>logging-panache</module>
```

Comment:
You need to add this new module to an existing slot in https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json

Reply:
Ah there are no native tests, maybe that's not necessary